lib/rspec/rails/adapters.rb

```diff
@@ -14,6 +14,12 @@ module Rails
       # Constant aliased to either Minitest or TestUnit, depending on what is
       # loaded.
       Assertions = Minitest::Assertions
+    elsif RUBY_VERSION >= '2.2.0'
+      # Minitest has been removed from ruby core. However, we must be on an
+      # old Rails version and must load the Minitest 4.x gem
+      gem 'minitest' if defined?(Kernel.gem)
```

Comment:
I am getting 
```
undefined method `message' for #<RSpec::Core::ExampleGroup::Nested_3::Nested_1:0x007fbb75fc4c38>
```
I am on Rails 4.1.15, Ruby 2.2.5 and RSpec 2.12.0 (Yes it is too old. I am migrating from Rails 3.2.22).
I have added minitest(5.9.0) and test-unit(3.2.0) both because RSpec won't run without them.
At some [places](https://github.com/rspec/rspec-rails/issues/773#issuecomment-20241398) it is written to remove test-unit, but then I won't be able to run RSpec at all. Any help ?
Also RSpec 2.12.0 is not a problem for Rails 4.1.15 yeah ?